### PR TITLE
add windows to github CI

### PR DESCRIPTION
**Pull request by fmoor**
Run tests on windows as well as linux.

### CODE REVIEW

**Comment by 1st1** [comment]
This is amazing, Frederick

**Review by 1st1 — APPROVED** [review]

**Comment by 1st1** [comment]
@jaclarke please also take a look

**Comment by jaclarke** [comment]
For some reason the deno tests are being skipped on windows

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -15,10 +15,18 @@ jobs:
     strategy:
       matrix:
         node-version: ["12", "13", "14", "15"]
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, windows-latest]
```
Comment (elprans) [inline .github/workflows/tests.yml:18]: Any reason to not also include macOS here?

Comment (fmoor) [reply .github/workflows/tests.yml:18]: No idea.

Comment (fmoor) [reply .github/workflows/tests.yml:18]: @1st1 or @jaclarke is there a reason that macOS should not be added?

Comment (1st1) [reply .github/workflows/tests.yml:18]: I think it should be

Comment (fmoor) [reply .github/workflows/tests.yml]: added

**Pull request merged**